Assets/Scripts/Functional Definitions/Interaction Definitions/SectorManager.cs

```diff
@@ -144,8 +144,8 @@ private void Update()
             {
                 dangerZoneTimer = 0;
                 Instantiate(damagePrefab, player.transform.position, Quaternion.identity);
-                player.TakeShellDamage(deadzoneDamage * player.GetMaxHealth()[0], 0, null);
-                player.TakeCoreDamage(deadzoneDamage * player.GetMaxHealth()[1]);
+                player.currentHealth[0] -= (deadzoneDamage * player.GetMaxHealth()[0]);
+                player.currentHealth[1] -= (deadzoneDamage * player.GetMaxHealth()[1]);
```

Comment:
This is a good workaround, but it might be better to have a direct mutator on currentHealth instead of using it like that. I think currentHealth should be private.

Reply:
> This is a good workaround, but it might be better to have a direct mutator on currentHealth instead of using it like that. I think currentHealth should be private.

done, mutator created but idk if there's any checks that need to be made for taking defense-piercing damage

Reply:
Try searching through references that modify currentHealth internally and change it to use the mutator. Also perhaps set currentHealth to private (it might already be)

Reply:
> Try searching through references that modify currentHealth internally and change it to use the mutator. Also perhaps set currentHealth to private (it might already be)

it was not private, so I set it to private and replaced all currentHealth modifications with mutator calls

Reply:
also, did I resolve the merge conflict correctly? I'm new to using github so i'm not really sure what i'm doing

Reply:
In C# methods generally start with capital letters

Reply:
fixed